### PR TITLE
prom_export: lint, docs updates, other chore/meta

### PR DESCRIPTION
**Pull request by tigattack**
# Initial Checklist
- [ ] Has @tigattack / @issy been added as a reviewer?
- [x] If applicable, have the relevant project(s), milestone(s), and label(s) been applied?
- [ ]  If applicable, have you added details of the cog to the readme as per [README.md](https://github.com/rHomelab/LabBot-Cogs/blob/main/README.md#cog-summaries)?

# Details
**Does this resolve an issue?**

Several undocumented :eyes:

## Changes

* Lint
* Update command docs & docstrings
* Add cog install message
* Add @rippleFCL to contributors
* Add cog requirements to info.json
* Add @tigattack to cog authors
* Rename cog (`prom_export` -> `prometheus_exporter`)

### Breaking Changes
* Base command changed (`prom_exporter` -> `prom_export`)

### CODE REVIEW

File: prom_export/info.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
     ],
     "description": "Exposes a HTTP endpoint for exporting guild metrics in Prometheus format",
     "short": "Exports guild metrics in Prometheus format",
-    "name": "Prom Exporter",
-
+    "name": "prom_export",
```
Comment (BeryJu) [inline prom_export/info.json:7]: technically `_exporter` would be more correct, in alignment with https://github.com/prometheus-community?q=exporter&type=all&language=&sort=

**Review by BeryJu — APPROVED** [review]

**Comment by rippleFCL** [comment]
Could i maby jump in on this? Theres quite a few changes i want to make to this cog if thats ok

**Comment by BeryJu** [comment]
Also the CI is failing because the markov cog is using `dict[str, ...]` which isn't supported in python 3.8, it was added in python 3.9

**Comment by rippleFCL** [comment]
@tigattack had a dense moment. Yh send the merg lad

**Pull request merged**